packages/core/src/html/siteAndPageNavProcessor.ts

```diff
@@ -38,18 +39,15 @@ const SITE_NAV_DROPDOWN_ICON_ROTATED_HTML = '<div class="site-nav-dropdown-btn-c
  * otherwise it is replaced with one until it is unique.
  */
 export class PageNavProcessor {
-  uuidTextNode: DomElement | undefined;
-  constructor() {
-    this.uuidTextNode = undefined;
-  }
+  uuidTextNode?: DomElement;
```

Comment:
Do we want to replace the type of this to be NodeOrText since parseHtml returns that?

Reply:
Nice catch! Amended to TextElement